### PR TITLE
Lross 4 public use

### PR DESCRIPTION
**Pull request by KjellKod**
With this pull request it is even easier to install StopWatch

Example of installation
```
cd MurMurHash
cd 3rdparty
unzip gtest-1.7.0.zip
cd ..
mkdir build
cd build
cmake ..
make -j
```

### Installing
```
sudo make install
```

Alternative on Debian systems
```
make package
sudo dpkg -i MurMurHash-<package_version>Linux.deb
```

### CODE REVIEW

File: CPackLists.txt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ #           LIBRARY DESTINATION lib/MurMurHash
+ #           COMPONENT libraries)
+
+ #  INSTALL( FILES ${HEADER_FILES}
+ #           DESTINATION include
+ #           COMPONENT headers)
+
+
+SET(MAJOR_VERSION 1)
+IF (NOT DEFINED ${VERSION})
```
Comment (KjellKod) [inline CPackLists.txt]: Versions don't have to be setup twice. Please keep the one in CMakeLists.txt

File: cmake/MurMurHashConfig.cmake [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#.rst:
+# FindMurMurHash
+# -------
+#
+# Find libMurMurhash, An implementation of the MurMurHash3 function in C++. 
+# MurmurHash3 was originally written by Austin Appleby and has been released into the public domain.
+#
+# Result variables
```
Comment (KjellKod) [inline cmake/MurMurHashConfig.cmake]: For this and the other LROSS pull requests: 

I do not think the <repo name>Config.cmake are necessary. Further, they are hard to test also so I believe all of the special cmake config files should no be installed and in fact removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)

File: CPackLists.txt [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          DESTINATION ${CPACK_INSTALL_PREFIX}/${CMAKE_INSTALL_HEADERDIR}
+          COMPONENT headers)
+
+
+SET(CPACK_COMPONENTS_ALL libraries headers)
+SET(CPACK_COMPONENT_LIBRARIES_DISPLAY_NAME "MurMurHash libraries")
+SET(CPACK_COMPONENT_HEADERS_DISPLAY_NAME "MurMurHash C++ headers")
+
+IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+   SET(CPACK_GENERATOR "DEB")
```
Comment (john-gress) [inline CPackLists.txt]: Combine this block with the one starting on line 63?

**Comment by john-gress** [comment]
👍

File: CPackLists.txt (hunk 1 above)
Comment (KjellKod) [inline CPackLists.txt]: done

File: CPackLists.txt (hunk 3 above)
Comment (KjellKod) [inline CPackLists.txt]: done

**Comment by john-gress** [comment]
👍

**Pull request merged**